site/components/case-study/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import InlineSVG from 'svg-inline-react';
+import styles from './style.css';
+
+/* PNG logo imports */
+import bbcPNG from './PNG/bbc.png';
+import bmwPNG from './PNG/bmw.png';
+import camdenPNG from './PNG/camden.png';
+import cartrawlerPNG from './PNG/cartrawler.png';
+import financialTimesPNG from './PNG/financialtimes.png';
+import fortnumPNG from './PNG/fortnum.png';
+import lloydsPNG from './PNG/lloyds.png';
+import selfridgesPNG from './PNG/selfridges.png';
+import skyPNG from './PNG/sky.png';
+import tescoPNG from './PNG/tesco.png';
+
+/* SVG figure imports */
+import tescoFigureSVG from './SVG/tesco-figure.svg';
+import fortnumFigureSVG from './SVG/fortnum-figure.svg';
+import camdenFigureSVG from './SVG/camden-figure.svg';
+import skyFigureSVG from './SVG/sky-figure.svg';
+
+const CaseStudy = () => (
+  <div className={styles.caseStudyContainer}>
+    <h2 className={styles.heading}>We solve complex problems and deliver real impact.</h2>
+    <div className={styles.figuresContainer}>
+      <div>
+        <InlineSVG src={fortnumFigureSVG} className={styles.caseFigure} />
+        <span className={styles.caseText}>Number of awards for the new online store</span>
+        <img alt="The logo of Fortnum & Masons" src={fortnumPNG} />
+      </div>
+      <div>
+        <InlineSVG src={camdenFigureSVG} className={styles.caseFigure} />
```

Comment:
I'm concerned about the accessibility loss of using SVGs for these numbers - do we have alt text? Can we just use real text and icons when needed?